diffpy/snmf/subroutines.py

```diff
@@ -81,13 +81,13 @@ def get_stretched_component(stretching_factor, component, signal_length):
     return np.interp(normalized_grid / stretching_factor, normalized_grid, component, left=0, right=0)
 
 
-def update_weights_matrix(component_amount, signal_length, stretching_factor_matrix, component_matrix, data_input,
-                          moment_amount, weights_matrix, method):
+def update_weights_matrix(number_of_components, signal_length, stretching_factor_matrix, component_matrix, data_input,
+                          number_of_moments, weights_matrix, method):
```

Comment:
moments is not the right word here.  IT is like the truncation of an expansion in moments or something.

How about `number_of_input_signals`?